anchovy_pasta_with_olives.md

```diff
@@ -0,0 +1,37 @@
+# anchovy pasta with olives
+
+## story
+I made this a few nights at Brahman 2020 at Pirates and it seemed relatively popular? I lived in Animal House but everybody cooked at Pirates so ya. I think I restocked anchovies at this grocery store at Niland we used to shop at? Maybe on the way back from Slab City. Because I had cooked this once and people seemed to like it and well, didn't want to mess with success. I also bought generic brand Spam over there and made myself spam and rice... yeah that recipe isn't worth posting. - @derekxkwan
+
+## general ingredients (in rough proportions)
+- half a jar of marinara sauce
+- half a can of pitted black olives
+- 2 tins of anchovies (because c'mon, one isn't enough)
+- package of pasta (spaghetti works, linguini also works)
+- 3-4 cloves of garlic (or garlic powder)
+- 3-4 roma tomatoes (because I am cheap and these are the cheapest)
+- smoked paprika (my cheat when I want something to have a lil' more depth)
+- dried oregano
+- dried basil (I can't remember if this was at Pirates)
+- salt
+- pepper
+- red pepper flakes (I can't remember if this was at Pirates either but I like adding red pepper flakes to everything)
+- whatever other spices are around (I think there was like this garlic/oregano powder mix laying around?)
+- capers (opt, I tend to make them without capers but I've seen recipes with capers)
+- some sort of cheese for topping optional (I tend to use parmesan but you do you, I think I brought some of the dry powdery stuff down)
+- cooking oil (I tend to use olive oil for these sorts of things but you do you)
+
+## directions
+1. boil your water for your pasta in a pot. you know, the normal amount of pasta boiling water.
+2. chop up your garlic cloves (if not garlic powder, you can't chop that up I don't think). i've found that cutting both ends off and then cutting through the middle of the clove lengthwise makes it easy to get peel off all the garlic stuff you usu don't want to cook.
```

Comment:
small tip: markdown automatically numbers your lists, so you can just have them as 

```md
1. blah
1. blah
1. etc
```

and it'll automatically renumber if you decide to add/remove a step in the middle